examples/render.rs

```diff
@@ -174,5 +176,21 @@ fn spawn_scene(
             Vec3::new(12.0, 12.0, 12.0),
             Vec3::new(0., 0., 0.),
             Vec3::Y,
-        ));
+        ))
+        .with_children(|builder| {
+            builder.spawn((
+                PbrBundle {
+                    mesh: meshes.add(Mesh::from(shape::Cube::new(1.0))),
+                    material: materials.add(Color::PURPLE.into()),
+                    transform: Transform::from_translation(Vec3::NEG_Z * 15.0),
+                    ..default()
+                },
+                TagCursor3d,
+                Name::new("Cursor3d"),
+            ));
+        });
 }
+
+#[derive(Component)]
+#[component(storage = "SparseSet")]
+pub struct TagCursor3d;
```

Comment:
What does this do?

Reply:
Good catch, thanks. 

That's just me having too many vim instances open and hitting the wrong key combo in one of them. Sorry!